src/app/routes/index.js

```diff
@@ -12,6 +12,10 @@ export const articleRegexPath = `/:service(${serviceRegex})/articles/:id(${idReg
 
 export const articleDataRegexPath = `${articleRegexPath}.json`;
 
+export const frontpageRegexPath = `/:service(${serviceRegex})`;
```

Comment:
I know this doesnt particularly matter for fixture, but I think were going to continue using `.amp` for our frontpages too :)